railties/lib/rails/generators/rails/app/templates/config/initializers/hsts_subdomains.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.action_dispatch.hsts_subdomains = true
```

Comment:
I don't get that is the problem. If we merged `{ expires: HSTS_EXPIRES_IN, subdomains: false, preload: false }` with `{ subdomain: true }` would not the result be `{ expires: HSTS_EXPIRES_IN, subdomains: true, preload: false }`?

```irb
>> default = { expires: 1, subdomains: false, preload: false }
=> {:expires=>1, :subdomains=>false, :preload=>false}
>> default.merge(subdomains: true)
=> {:expires=>1, :subdomains=>true, :preload=>false}
```